core/controllers/topic_editor.py

```diff
@@ -151,6 +152,23 @@ def get(self, topic_id: str) -> None:
 
         updated_canonical_story_summary_dicts = []
         for summary in canonical_story_summary_dicts:
+            story = story_fetchers.get_story_by_id(summary['id'])
```

Comment:
Don't do a get_by_id in a for loop -- that's an antipattern and will result in long lags for users. You should be doing a single get_multi() call instead in this method.

Reply:
Done